python/pyspark/sql/tests.py

```diff
@@ -48,12 +49,12 @@
 else:
     import unittest
 
+from pyspark.util import _exception_message
```

Comment:
nit: add an empty line between this import and `_pandas_requirement_message` line.